tasks/rh-sign-image/rh-sign-image.yaml

```
@@ -54,19 +93,79 @@ spec:
             echo "No valid data file was provided."
             exit 1
         fi
+        RPA_FILE="$(workspaces.data.path)/$(params.releasePlanAdmissionPath)"
+        if [ ! -f "${RPA_FILE}" ] ; then
+            echo "No valid rpa file was provided."
+            exit 1
+        fi
+
+        REQUESTTYPE=$(jq -r '.sign.requestType // "internal-request"' "${DATA_FILE}")
+        service_account_name=$(jq -r '.spec.pipeline.serviceAccountName // "appstudio-pipeline"' "${RPA_FILE}")
+        if [ "${REQUESTTYPE}" == "internal-pipelinerun" ] ; then
+          requestType=internal-pipelinerun
+          serviceAccountName=${service_account_name}
+          taskGitUrl=$(params.taskGitUrl)
+          taskGitRevision=$(params.taskGitRevision)
+          defaultRequest="simple-signing-pipeline"
+        else
+          requestType=internal-request
+          # we do not need to set these for internal-requests
+          serviceAccountName=
+          taskGitUrl=
+          taskGitRevision=
+          defaultRequest="hacbs-signing-pipeline"
+        fi
+        # shellcheck disable=SC2086
+        request=$(jq --arg defaultRequest "${defaultRequest}" -r '.sign.request // $defaultRequest' ${DATA_FILE})
```

Why the shellcheck ignore? Isn't this as easy as enclosing `${DATA_FILE}` in `""`?
```suggestion
        request=$(jq --arg defaultRequest "${defaultRequest}" -r '.sign.request // $defaultRequest' "${DATA_FILE}")
```

---

I think I thought they were talking about `$defaultRequest` :)

I'll fix it.

---

fixed